index.html

```diff
@@ -1,167 +1,203 @@
+            console.log(canvasHeight);
+            paper.setSize(Math.round($(window).width()) + 'mm', canvasHeight);
+
+            for (var i = 0; i < settings.signs.length; i++) {
+                const sign = settings.signs[i].trim();
+                createSign(sign, i * spacing, settings);
             }
 
-            function generateSigns() {
-              try {
-                const settings = {
-                  signs: $("#csv").val().split(','),
-                  padding: parseInt($('[name="padding"]').val(), 10),
-                  border: parseInt($('[name="border"]').val(), 10),
-                  fontSize: parseInt($('[name="fontsize"]').val(), 10),
-                  fontFamily: $('[name="fontfamily"]').val(),
-                };
-                if (settings.fontFamily != "Maven Pro") {
-                  throw new Error('Font family must be set to Maven Pro (other fonts are not yet implemented)');
-                }
-
-                const signHeight = Math.round(2*settings.padding + settings.fontSize);
-                const spacing = 20 + signHeight;
-                paper.clear();
-
-                const canvasHeight = Math.round(1.1*spacing*settings.signs.length+100) + 'mm';
-                console.log(canvasHeight);
-                paper.setSize(Math.round($(window).width())+'mm', canvasHeight);
-
-                for (var i = 0; i < settings.signs.length; i++) {
-                  const sign = settings.signs[i].trim();
-                  createSign(sign, i*spacing, settings);
-                }
-
-                $("#stats").html(`
+            $("#stats").html(`
                   <div>Generated ${settings.signs.length} signs</div>
                   <div>Sign heights are ${signHeight}mm</div>
                 `)
-              } catch(e) {
-                alert(e);
-              }
-            }
+        } catch (e) {
+            alert(e);
+        }
+    }
 
-            // Setup user actions
-            $("input,textarea").on('blur', generateSigns);
-            $("input,textarea").on('keypress', (event) => {
-              if (event.which === 13) {
-                generateSigns();
-              }
-            });
-            $("#generate").click(generateSigns);
-            $('#download').click(() => {
-              var serializer = new XMLSerializer();
-              var svgString = serializer.serializeToString(paper.canvas);
-              console.log(svgString);
-              a = document.createElement('a');
-              a.download = 'render.svg';
-              a.type = 'image/svg+xml';
-              blob = new Blob([svgString], {"type": "image/svg+xml"});
-              a.href = (window.URL || webkitURL).createObjectURL(blob);
-              a.click();
-            });
-          });
-        </script>
-    </body>
+    // Setup user actions
+    $("input,textarea").on('blur', generateSigns);
+    $("input,textarea").on('keypress', (event) => {
```

Comment:
L178-200 indentation/line separation is a bit weird IMO. Could you please update this to match the style in the rest of the doc?